databroker/broker.py

```diff
@@ -232,24 +230,25 @@ def _(key, mds):
         raise ValueError("key=%r matches %d runs. Provide "
                          "more characters." % (key, len(results)))
     result, = results
-    header = Header.from_run_start(mds, result)
-    return header
+    return [(result, _safe_get_stop(db.mds, result))]
 
 
 @search.register(set)
 @search.register(tuple)
 @search.register(MutableSequence)
-def _(key, mds):
+def _(key, db):
     logger.info('Interpreting key = {} as a set, tuple or MutableSequence'
                 ''.format(key))
-    return [search(k, mds) for k in key]
+    return sum((search(k, db) for k in key), [])
 
 
-class Broker(object):
-    def __init__(self, mds, fs, plugins=None, filters=None):
+class BrokerES(object):
+    def __init__(self, hs, es, plugins=None, filters=None):
         """
         Unified interface to data sources
 
+        Eventually this API will change to ``__init__(self, hs, es, **kwargs)``
```

Comment:
The only place plugins ever was used was CSX, and I don't think it is used actively there. Safe to remove `plugins` now if you want, especially since archiver integration is broken.